src/modules/StudentsTable/components/Dashboard/components/TableBody/index.tsx

```diff
@@ -1,29 +1,37 @@
 import React, { FC, useMemo, ReactText } from 'react';
+import { useSelector } from 'react-redux';
 import { FixedSizeList as List } from 'react-window';
 import AutoSizer from 'react-virtualized-auto-sizer';
 import { StyledTableBody } from './styled';
 import './styles.css';
 import { User, Course, Team } from 'types';
 import { USERS_PER_PAGE } from 'appConstants';
 import { TableRow } from './components/TableRow';
+import { selectCurrCourse } from 'modules/LoginPage/selectors';
 
 type TableBodyProps = {
   users: User[];
   page: number;
 };
 
 export const TableBody: FC<TableBodyProps> = ({ users, page }) => {
+  const currCourse = useSelector(selectCurrCourse);
+
   const usersData: Array<string[] | ReactText[]> = useMemo(
     () =>
       users.map((user: User, index: number) => {
         return [
           `${index + 1 + page * USERS_PER_PAGE}`,
           `${user.firstName} ${user.lastName || null}`,
           user.score,
-          user.teams.length
-            ? (user.teams as Array<Team>)
-                .map((team: Team) => `${team.number}`)
-                .join(', ')
+          (user.teams as Team[]).find(
```

Comment:
do we still need this extra description? `(user.teams as Team[])`

Reply:
It we have problem - we should may be remove |[] from User type teams field description

Reply:
Done